docs/concepts/what-is-a-workflow-execution.md

```diff
@@ -2,57 +2,123 @@
 id: what-is-a-workflow-execution
 title: What is a Workflow Execution?
 sidebar_label: Workflow Execution
-description: A Workflow Execution is a Reentrant Process; that is, a resumable, recoverable, and reactive process.
+description: A Temporal Workflow Execution is a durable, scalable, reliable, and reactive function execution. It is the main unit of execution of a Temporal Application.
 tags:
   - explanation
 ---
 
-A Workflow Execution is a Reentrant Process; that is, a resumable, recoverable, and reactive process:
+A Temporal Workflow Execution is a durable, reliable, and scalable function execution.
+It is the main unit of execution of a [Temporal Application](/docs/concepts/what-is-a-temporal-application).
 
-- Resumable: Ability of a process to continue execution after execution was suspended on an await-able.
-- Recoverable: Ability of a process to continue execution after execution was suspended on a failure.
-- Reactive: Ability of a process to react to external events.
+- [How to spawn a Workflow Execution in Go](/docs/go/how-to-spawn-a-workflow-execution-in-go)
+- [How to spawn a Workflow Execution in Java](/docs/java/how-to-spawn-a-workflow-execution-in-java)
+- [How to spawn a Workflow Execution in PHP](/docs/php/workflows/#starting-workflows)
+- [How to spawn a Workflow Execution in TypeScript](/docs/typescript/workflows#how-to-start-and-cancel-workflows)
+
+Each Temporal Workflow Execution has exclusive access to its local state, executes concurrently to all other Workflow Executions, and communicates with other Workflow Executions via [Signals](/docs/concepts/what-is-a-signal) and the environment via [Activities](/docs/concepts/what-is-an-activity). While a single Workflow Execution has limits on size and throughput, a Temporal Application can consist of millions to billions of Workflow Executions.
+
+**Durability**
+
+Durability is the absence of an imposed time limit.
+A Workflow Execution is durable because it executes a Temporal Workflow Definition, also called a Temporal Workflow Function, your application code, effectively once and to completion—whether your code executes for seconds or years.
```

Comment:
```suggestion
A Workflow Execution is durable because it executes a Temporal Workflow Definition (also called a Temporal Workflow Function), your application code, effectively once and to completion—whether your code executes for seconds or years.
```

Reply:
You could also break it down into two sentences so the idea of durability is not lost in additional info. For example: "A Workflow Execution is durable because it executes a Temporal Workflow Definition effectively once and to completion—whether your code executes for seconds or years. The Temporal Workflow Definition is your application code , and is also called a Temporal Workflow Function." 

Or remove the second sentence "The Temporal Workflow Definition is your..." and place it within the "What is a Workflow Definition" and link the Workflow Definition here.